superflore/generators/ebuild/run.py

```diff
@@ -31,12 +32,16 @@
 from superflore.utils import info
 from superflore.utils import load_pr
 from superflore.utils import ok
+from superflore.utils import ros2_distros
 from superflore.utils import save_pr
 from superflore.utils import url_to_repo_org
 from superflore.utils import warn
 
-# Modify if a new distro is added
-active_distros = ['indigo', 'kinetic', 'lunar']
+# TODO(allenh1): It would be super nice make this a configuration option.
```

Comment:
:+1:  `ROSDISTRO_INDEX_URL`

Just use `rosdistro.get_index_url()` https://github.com/ros-infrastructure/rosdistro/blob/master/src/rosdistro/__init__.py#L68

Reply:
I figured there was going to be something like that. I'll look into this in a follow up. Thanks.